fluent-bit-to-loki/pkg/client/client_test.go

```diff
@@ -0,0 +1,91 @@
+package client
```

Comment:
Shouldn't it be ?
```suggestion
package client_test
```

Reply:
newTimestampOrderingClient is not exposed function,